app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,54 @@
+class FeedbacksController < ApplicationController
+  layout 'full_width'
+
+  def new
+    @booking_feedback = BookingFeedbackForm.new
+  end
+
+  def create
+    @booking_feedback = BookingFeedbackForm.new(booking_feedback_params)
+
+    Feedback.create_ticket(@booking_feedback.message_content) if @booking_feedback.valid?
+
+    if request.xhr?
+      render_xhr_response
+    else
+      render_html_response
+    end
+  end
+
+  def thanks
+  end
+
+  private
+
+  def render_xhr_response
+    if @booking_feedback.valid?
+      render_result :ok
+    else
+      render_result :bad_request
+    end
+  end
+
+  def render_html_response
+    if @booking_feedback.valid?
+      redirect_to thanks_feedback_path
+    else
+      render :new
+    end
+  end
+
+  def render_result(status)
+    render partial: 'result', status: status if request.xhr?
```

Comment:
Is this guard still needed? Looks like this is only called from `render_xhr_response`